tests/examples/source/devfiles/nodejs/devfile-with-multiple-projects.yaml

```diff
@@ -0,0 +1,41 @@
+schemaVersion: 2.0.0
+metadata:
+  name: nodejs
+projects:
+  - name: nodeshift
+    clonePath: webapp/
+    git:
+      remotes: "https://github.com/nodeshift/nodeshift"
+  - name: nodejs-health-check
+    git:
+      remotes: "https://github.com/nodeshift-starters/nodejs-health-check" 
```

Comment:
I am facing problems with the schema in this file
```
[mrinaldas@localhost project]$ odo push
 ✗  invalid devfile schema. errors :
- projects.0.git.remotes: Invalid type. Expected: object, given: string
- starterProjects.0.git.remotes: Invalid type. Expected: object, given: string
```
It should be 
```
  git:
      remotes:
        origin: "https://github.com/odo-devfiles/nodejs-ex.git"
```

Reply:
ohh!! sorry updated